src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  tooltip: {
+    background: 'rgba(0, 0, 0, .8)',
+    color: 'white',
+    borderRadius: '4px',
+    pointerEvents: 'none',
+  },
+  tooltipKey: {
+    display: 'inline-block',
+    width: '10px',
+    height: '10px',
+    marginRight: '10px',
+  },
+};
+
+function Footer({ dataset, index, currData }) {
+  const prevData = dataset[index - 1].value;
+  const prevRevision = dataset[index - 1].revision;
```

Comment:
instead of `prevData` and `prevRevision`

    const prev = dataset[index - 1]

and refer to `prev.value` and `prev.revision`

Reply:
...and use destructuring

    const [prev, curr] = series[datasetIndex].data.slice(index - 1);